MangaRipper/Forms/FormMain.cs

```diff
@@ -21,6 +22,26 @@ public partial class FormMain : Form
         private BindingList<DownloadChapterTask> _downloadQueue;
         private readonly ApplicationConfiguration _appConf = new ApplicationConfiguration();
 
+        private string SaveDestination
+        {
+            get
+            {
+                if (rdSeriesDestination.Checked)
+                    return lbSeriesDestination.Text;
```

Comment:
"Visual StyleCop" also a good additional for VS
![image](https://cloud.githubusercontent.com/assets/17333840/24578259/745550e2-16fe-11e7-8206-df91227ef7d6.png)